app/src/main/java/com/example/taskyy/data/room_entity/UserEntity.kt

```diff
@@ -1,14 +1,15 @@
 package com.example.taskyy.data.room_entity
 
+import androidx.room.ColumnInfo
 import androidx.room.Entity
 import androidx.room.PrimaryKey
 
 @Entity(tableName = "user_table")
 data class UserEntity(
     @PrimaryKey(autoGenerate = true)
-    val id: Int,
-    val firstName: String,
-    val lastName: String,
-    val phoneNumber: String
+    @ColumnInfo val id: Int,
+    @ColumnInfo val firstName: String,
+    @ColumnInfo val lastName: String,
+    @ColumnInfo val phoneNumber: String
```

Comment:
These annotations aren't necessary, by default Room will use the variable name as column name